cypress/support/commands.js

```diff
@@ -120,10 +103,15 @@ Cypress.Commands.add("datasetMeetsTestCriteria", (p_pageName, p_datasetConfig, p
 // Calls mutation in the Nuxt store
 Cypress.Commands.add("commitToVuexStore", (p_mutation, p_data) => {
 
+    console.log(`In commitToVuexStore for ${p_mutation} with data: ${JSON.stringify(p_data)}`);
```

Comment:
This looks like a leftover debug statement.
If it is intentional, I think we can achieve the same thing more cleanly by using the vue-dev-utils in the browser

Reply:
All `console.log` lines in the project should be removed. They are for debug purposes only.